### DISCUSSION TITLE
编译报错

### DISCUSSION THREAD

**Discussion by lei1320719647** [note286/xduts#15] [Q&A] [answered]
编译中会出现下面的警告，然后编译就卡住了，请问该怎么解决

xdupgthesis.cls

LaTeX Warning: You have requested package `', but the package provides `xdupgthesis'.

**Comment by note286** [comment]
无法复现你的问题，请描述更详细些。

**Comment by lei1320719647** [comment]
编译过程中会出现下面的问题：
LaTeX Warning: You have requested package ', but the package provides xdupgthesis'.
![image](https://user-images.githubusercontent.com/55944911/198971681-b3a3ca0c-a12c-4144-ae76-c76ac0601462.png)
然后会一直保持这种状态，请问该怎么解决呢

**Comment by note286** [comment]
请问你在编译什么文件，给出你的编译命令和编译的文件名。

**Comment by lei1320719647** [comment]
文件是xdupgthesis.cls
编译命令：xelatex xdupgthesis.cls

**Comment by note286** [comment, marked as answer]
你阅读一下文档`xduts.pdf`吧，你要编译`.tex`文件，这是你自己建立的文件。

**Comment by lei1320719647** [comment]
解决了谢谢
